### PR TITLE
updated code to run with Python 3

### PR DESCRIPTION
**Pull request by faisalmlalani**
@bderenzi ready for review!

### CODE REVIEW

File: outlierdetect/__init__.py [hunk 1]
```diff
@@ -12,5 +12,5 @@
 # or implied.  See the License for the specific language governing permissions and limitations
 # under the License.
 
-
-from outlierdetect import run_sva, run_mma, __doc__
+# Leading to a circular import, so commented out.
+# from outlierdetect import run_sva, run_mma, __doc__
```
Comment (bderenzi) [inline outlierdetect/__init__.py]: This is not the correct solution. More details here:

https://docs.python.org/3/reference/import.html#regular-packages

Correct code should be:
```python
from .outlierdetect import run_sva, __doc__
# run_mma will be missing when stats are not available
try:
    from .outlierdetect import run_mma
except ImportError:
    pass
```

I've tested this on 3.8.6 and 2.7 and it works on both

File: example.py [hunk 2]
```diff
@@ -9,8 +9,9 @@
 Example use of outlierdetect.py module.
 """
 
+from __future__ import print_function
 from matplotlib import mlab
-import outlierdetect
+from outlierdetect import outlierdetect
```
Comment (bderenzi) [inline example.py]: Can revert once the change is made to `__init__.py`

**Review by bderenzi — CHANGES_REQUESTED** [review]
Looks good. One small fix on the module stuff, but once that's sorted we're good to go, I think

**Comment by faisalmlalani** [comment]
@bderenzi fixed!

**Review by bderenzi — APPROVED** [review]
Looks good -- thanks

**Pull request merged**